### DISCUSSION TITLE
Welcome to Inertia.js Discussions!

### DISCUSSION THREAD

**Discussion by claudiodekker** [inertiajs/inertia#358] [Show and tell]
## 👋  Welcome to Inertia.js Discussions!

In case you're not yet familiar with the Discussions tab, the idea here is to have a place to:
- Ask questions.
- Share ideas.
- [Show off your own things](https://github.com/innocenzi/awesome-inertiajs).

For more casual and realtime conversations with the rest of the community, we recommend you to [visit our Discord community instead](https://discord.gg/gwgxN8Y), but we hope this helps to differentiate actual [known Issues](https://github.com/inertiajs/inertia/issues) from questions and such!

**Comment by reinink** [comment]
Thanks for getting this setup @claudiodekker! 🙌
